### ISSUE TITLE
broken relative path to voodoospark.cpp

### ISSUE THREAD

**Issue by noopkat** [nodebotanist/voodoospark-installer#1]
Hello!

Here is an error I get when it tries to flash:

```
{ [Error: form-data: ENOENT, open '<the local dir I was in>/usr/local/lib/node_modules/voodoospark-installer/firmware/voodoospark.cpp']
  errno: -2,
  code: 'ENOENT',
  path: '<the local dir I was in>/usr/local/lib/node_modules/voodoospark-installer/firmware/voodoospark.cpp' }
```

`<local dir I was in>` was just the directory I was currently sitting in when I ran the `voodoospark` command.

If I may, I would suggest [on the following line](https://github.com/nodebotanist/voodoospark-installer/blob/master/bin/index.js#L53) changing it to something a little more robust, like:

`path.resolve(__dirname, '..', 'firmware', 'voodoospark.cpp')`

Everything else is super nice so far, the UX is A+ :100:

**Comment by noopkat** [comment]
I found this, not sure if it might be causing the issue? https://github.com/spark/sparkjs/blob/master/lib/spark-api.js#L344

**Comment by k45514n** [comment]
Yep, that's definitely doing it. Let me see what I can do for a workaround haha.

**Comment by k45514n** [comment]
I think we got it in version 0.1 thanks to your footwork!

addressed in f7597672f99ed9839d415c4522bdd90c2200d406, ready for testing.

**Comment by noopkat** [comment]
Nailed it! It's totally working breezy now :100:
